schedule/yast/sle/usb_install.yaml

```diff
@@ -8,6 +8,7 @@ vars:
   USBBOOT: 1
   YUI_REST_API: 1
 schedule:
+  default_systemd_target: []
```

Comment:
no need to overwrite as it will not be present in default.

Reply:
we should not edit this file, as it is used in Online medium, just create a new one with the overwrite for skip_registration. We can tackle in the future how to keep single test suite if worth it.

Reply:
you forgot the overwrite.